### PR TITLE
Version 4.4

### PR DESCRIPTION
**Pull request by Ericgig**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Ericgig** [comment]
@conda-forge-admin, please rerender

**Comment by ajgpitch** [comment]
@Ericgig I just merged #36 . The Windows builds are working in that. 
I think if you now merge the master into this, then overwrite the `meta.yaml` with this one, and bump the build number, that all could be well.

**Comment by Ericgig** [comment]
@conda-forge-admin, please rerender

**Comment by ajgpitch** [comment]
@conda-forge-admin, please rerender

**Comment by ajgpitch** [comment]
#36 the automated build seemed to build the Windows package for Python 3.x fine. We merged this into this branch, and so were expecting this to work here, but we are getting
```
Cmd.exe exited with code '1'.
```
Also not sure why we are getting a VS2008 build when we have specified `python >= 3`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,30 +10,29 @@ source:
...
     #v4.2.0 build 2: sha256: 87d677c6450a17a4dd8cfa1c7ae5f1f7646c47b63dab5abd88980093a47611e5
+    #v4.3.1 build 2: sha256: e2d243a24b5b8577337cfc57fe1efbe12cb3f4a0d6e14c06aa9583dbe68d53f4
 
 build:
-    number: 0
+    number: 1
     script: python -m pip install --no-deps --ignore-installed . --verbose  # [unix]
-    # ajgpitch 2017-03-16: win py27 build currently failing due to error on import scipy.misc
-    # ajgpitch 2017-07-28: decided to give up on win py2k build until further notice
-    skip: True                        # [win and py2k]
```
Comment (xhochy) [inline recipe/meta.yaml]: You should readd this skip to get rid of the the Windows Python 2 build

**Comment by ajgpitch** [comment]
@conda-forge-admin, please rerender

**Comment by ajgpitch** [comment]
@conda-forge-admin, please rerender

**Pull request merged**